torch_xla/csrc/ops/unique2.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "torch_xla/csrc/ir.h"
+
+namespace torch_xla {
+
+class Unique2 : public XlaNode {
+ public:
+  Unique2(const torch::lazy::Value& input);
+
+  torch::lazy::NodePtr Clone(torch::lazy::OpList operands) const override;
+
+  XlaOpVector Lower(LoweringContext* loctx) const override;
```

Comment:
nit: mind adding a ToString()?

Reply:
It appears ToString() override is only needed when the op has input types other than lazy::Value. So I guess the default should be fine.